.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

```suggestion
    branches: *
```

This should run on each PR so we know if something is breaking before getting it merged